pkg/controller/master/supportbundle/manager.go

```diff
@@ -90,6 +90,14 @@ func (m *Manager) Create(sb *harvesterv1.SupportBundle, image string, pullPolicy
 									Name:  "SUPPORT_BUNDLE_NAME",
 									Value: sb.Name,
 								},
+								{
+									Name:  "SUPPORT_BUNDLE_DESCRIPTION",
+									Value: sb.Spec.Description,
```

Comment:
Hi Jack, can you test if it works if the description contains some characters like `'`, `"`, `\`? Thanks!

Reply:
Sure!

env
![image](https://github.com/harvester/harvester/assets/6960289/8a283e6d-b82f-4682-a7d2-3463f3f1c1a0)
![image](https://github.com/harvester/harvester/assets/6960289/331ae2a8-2a61-4fe7-b0fd-9d27f93ae140)


downloaded metadata.yaml
![image](https://github.com/harvester/harvester/assets/6960289/0a2a44a4-d70f-400d-9f1d-ff56074707f5)
![image](https://github.com/harvester/harvester/assets/6960289/7bfc0bd0-c3ee-4d22-9349-a1ff83c836f7)